ape_safe/client/mock.py

```diff
@@ -109,3 +109,8 @@ def post_signatures(
                     signatureType=SignatureType.EOA,
                 )
             )
+
+    def estimate_gas_cost(
+        self, receiver: AddressType, value: int, data: bytes, operation: int = 0
+    ) -> Optional[int]:
+        return None  # Estimate gas normally
```

Comment:
I'm assuming this works in the tests, but if not I think the issue is likely that the gas prices are nonzero

Reply:
What issue?